definitions/variable/sdg-indicators/sdg.yaml

```
@@ -14,6 +14,22 @@
       below USD_2011 5.50 PPP per day (World Bank LMIC definition)
     sdg: 1
     unit: million
+- Population|Risk of Hunger:
+    definition: Population at risk of hunger, calculated by multipling total population
+      and prevalence of undernourishment which is computed from a probability distribution
+      of daily dietary energy consumption and minimum dietary energy requirement.
+    unit: million
+- Population|Electricity Access:
+    definition: Population with access to electricity
+    unit: million
+- Population|Relying on Solid Fuels:
+    definition: Population without access to electricity and relying on solid fuels
+      for heating-cooking
+    unit: million
+- Population|Clean cooking Access:
+    definition: Population with access to improved cooking facilities (electric or
+      gas)
+    unit: million
```

Apply spelling conventions and add reference to previous variable name
```suggestion
- Population|Clean Cooking Access:
    definition: Population with access to improved cooking facilities (electric or
      gas)
    unit: million
    navigate: Population|Clean cooking Access
```